modules/slugs/admin/slugs-admin.php

```diff
@@ -7,8 +7,16 @@
 // add filters
 add_filter( 'wp_get_object_terms', 'qtranxf_slugs_get_object_terms', 0, 4 );
 add_filter( 'get_terms', 'qtranxf_slugs_get_terms', 0, 3 );
+
 // admin actions
-add_action( 'add_meta_boxes', 'qtranxf_slugs_add_slug_meta_box' );
+
+// add slug metabox
+$curr_post_type = $_GET[ 'post_type' ] ? $_GET[ 'post_type' ] : get_post_type( intval( $_GET[ 'post' ] ) );
+if ( $curr_post_type !== 'shop_order' && $curr_post_type !== 'shop_coupon' )
```

Comment:
Is that enough to read the `$_GET` values?

Reply:
no, ref next comment